src/bin/mqtt.ts

```diff
@@ -7,10 +7,12 @@
  * See LICENSE for more information
  */
 import path from 'path'
+// @ts-expect-error - There is no types available for this library.
 import Commist from 'commist'
+// @ts-expect-error - There is no types available for this library.
 import help from 'help-me'
-import publish from './pub'
-import subscribe from './sub'
+import publish from './pub.js'
+import subscribe from './sub.js'
```

Comment:
Best practice to avoid confusion between folders and files, there are also other more specific requirements from Node.js and browsers if using `.mjs` extension.